lib/aws_client_wrapper.rb

```diff
@@ -15,7 +15,8 @@ def role_credentials
       @role_credentials ||= Aws::AssumeRoleCredentials.new(
         client: core_aws_client,
         role_arn: role_arn,
-        role_session_name: role_session_name
+        role_session_name: role_session_name,
+        external_id: "dalmatian-tools"
```

Comment:
Asked bob about hardcoding this and it was deemed good enough now because making it something different would generate a lot of extra work.